### PR TITLE
Added documention for list and input component

### PR DESCRIPTION
**Pull request by DevRaahul**
1.  Implemented suggested changes from last PR.
2. Added docs for list and input component
3. Added class for code-container

1. Input screen - 
image
2.List Screen- 
https://drive.google.com/file/d/1eEwn0iApVz11c-i64z0fPuA-LhDJdAms/view

### CODE REVIEW

File: Input/index.html [hunk 1]
```diff
@@ -7,21 +7,95 @@
...
+                    class="code-sandbox" sandbox="allow-scripts allow-same-origin">
+                </iframe>
+            </div>
         </div>
     </div>
+
+    <script src="../home.js" />
 </body>
 
 </html>
```
Comment (VishalPatil18) [inline Input/index.html:101]: You could add an empty line at the end to avoid such warnings by github.

**Review by VishalPatil18 — APPROVED** [review]
Just a suggestion added. Apart from that the code looks good and works well. Great work @DevRaahul 👍

File: Input/index.html (hunk 1 above)
Comment (DevRaahul) [reply Input/index.html:101]: Yes sure.

**Review by abutaher11 — APPROVED** [review]
Code looks good to merge

**Pull request merged**